### ISSUE TITLE
File save isn't saving to the local file

### ISSUE THREAD

**Issue by tinamayswinkels** [adobe/brackets#12382]
We've had this issue on numerous computers, where clicking file save or hitting command s doesn't actually csave the changes to the source file. Can't identify what triggers the behaviour because it doesn't happen all the time. 

 https://www.dropbox.com/s/dugyu6cxttf370d/bug%20-%20adobe.mov?dl=0

**Comment by jdavies322** [comment]
Yep. I also have experienced this. Make changes to a file, hit save. Write to disk appears to not actually happen. No error message. The indicator that the file has been modified does not go away. The main tell is that browser refresh does not occur, nor any stack watch processes. Neither keyboard commands or context menu Save seems to work. This has only happened twice in the couple months I've been working with Brackets. Haven't determined a pattern to the failure. Quitting Brackets and relaunching seems to get things back in order.

Brackets Release 1.6 build 1.6.0-16680.

Mac OS X 10.9.5.

Extensions:
adobe.brackets.extract 0.8.0-1749
brackets-emmet 1.2.2
bracketstoix-3.2.0
gruehle.exclude-folders 0.1.0

**Comment by LBTimer** [comment]
To me, it happens like this:

1) I have one file open in Brackets editor.
2) I select "Save As" and give a different name for the file. A copy of the file is created (can be seen in the folder tree on the left) but the editor remains in the original file
3) If I make any change in the original file and try to save (File - Save or Ctrl+S), it doesn't save.

Release 1.6 experimental build 1.6.0-16680
Linux Mint 17.2

**Comment by kowsoleea** [comment]
I have this issue every so often (Linux Mint 17.3). I notice that Less Autocompile doesn't compile the file after saving. However, if I change to antother file (i.e. with Alt-Tab) and back to my Less file the file is saved and compiled. I find it a bit annoying, but it doesn't happen all the time. Sometimes it helps if I change the project folder, and back into the original project. Sometimes restarting Brackets helps. So far I couldn't find a pattern.

This also (used to) happen under Windows, although I didn't see it after changing to 1.7 under Windows. Under Linux the 1.7 seems the same. Sometimes it saves, sometimes it doesn't. Maybe, if someone could point me to some logfile or console I could help to pinpoint the problem.

regards, Ivan
